projects/RPi/options

```diff
@@ -12,6 +12,9 @@
           TARGET_CPU="arm1176jzf-s"
         elif [ "$DEVICE" = "RPi2" -o "$DEVICE" = "Slice3" ]; then
           TARGET_CPU="cortex-a7"
+        elif [ "$DEVICE" = "RPi4" ]; then
+          TARGET_CPU="cortex-a53"
```

Comment:
TARGET_CPU="cortex-a72" ? Will need config/arch.arm updated too.

Reply:
I've been testing with cortex-a72 but that lead to gdb spinning forever on a backtrace (eg when crashlog is created or when manually doing "thread apply all bt")
```
      Thread 15 (Thread 0xa51bc370 (LWP 539)):
      #0  futex_abstimed_wait_cancelable (private=0, abstime=0xa51bb930, expected=0,
          futex_word=0x3884df8) at ../sysdeps/unix/sysv/linux/futex-internal.h:205
      #1  __pthread_cond_wait_common (abstime=0xa51bb930, mutex=<optimized out>,
          cond=0x3884dd0) at pthread_cond_wait.c:539
      #2  __pthread_cond_timedwait (cond=0x3884dd0, mutex=0x0, abstime=0xa51bb930)
          at pthread_cond_wait.c:667
      #3  0x008b6130 in ?? ()
      #4  0x008bd58c in ?? ()
      #5  0x008219ac in XBMCAddon::xbmc::Monitor::waitForAbort(double) ()
      #6  0x0084fbac in ?? ()
      #7  0xb6c0a018 in PyEval_EvalFrameEx () from /usr/lib/libpython2.7.so.1.0
      #8  0xb6c0a018 in PyEval_EvalFrameEx () from /usr/lib/libpython2.7.so.1.0
      #9  0xb6c0a018 in PyEval_EvalFrameEx () from /usr/lib/libpython2.7.so.1.0
```

As we didn't have the time to investigate this issue further I went back to cortex-a53 which didn't show this behaviour.

But I agree, we will need to revisit this at some point and if the issues are solved we can switch to cortex-a72

Reply:
TARGET_CPU_FLAGS="+crc" should go in here too.

Reply:
TARGET_CPU_FLAGS="+crc" is already there, in the next line :-)